plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestingMySqlServer.java

```diff
@@ -105,7 +105,7 @@ public String getDatabaseName()
 
     public String getJdbcUrl()
     {
-        return format("jdbc:mysql://%s:%s?useSSL=false&allowPublicKeyRetrieval=true", container.getHost(), container.getMappedPort(MYSQL_PORT));
+        return format("jdbc:mysql://%s:%s?useSSL=false&allowPublicKeyRetrieval=true&zeroDateTimeBehavior=convertToNull", container.getHost(), container.getMappedPort(MYSQL_PORT));
```

Comment:
Revert. It would be better to avoid changing the default variable as much as possible.